demosplan/DemosPlanCoreBundle/Resources/client/scss/components/_statement.scss

```diff
@@ -170,7 +170,7 @@
             }
         }
 
-        a.display--none {
+        a.hidden {
             display: none;
         }
```

Comment:
can't we remove this since it comes as utility from tailwind?

Reply:
i'm pretty sure this is some "stupid but important" specificity hack - in the statement modal there are plenty `nojs-display--block` things which mess around with specificity.

However as i could not even track down [the dark origin of this code](https://github.com/demos-europe/demosplan/commit/ac73f863ec33c21253eef46a0b08323a3b4934a7), i removed it! 03a24ce2711f8c5135d6b5c5e325f1f92aaadfcb